corehq/apps/hqwebapp/models.py

```diff
@@ -53,6 +53,23 @@ def get_latest_alert(cls):
             return ''
 
 
+class UserAgent(models.Model):
+    MAX_LENGTH = 255
```

Comment:
I'm willing to be convinced otherwise, but my impression is that we're not losing anything of value by truncating user agent strings to 255. For example, those extra numbers on the one you linked don't really add anything to our log entries.  These values are stored so that in the future we can display to the user all his attempted logins, in the case that the user is concerned his account is being hacked or was compromised. To me, the only thing I care about in that use case is if he was using the same browser as me -- which I can determine in 255 chars.